README.md

```diff
@@ -228,7 +244,7 @@ node module. To install it, make sure you have install the dev dependencies from
 the table of contents, run:
 
 ```shell
-./node_modules/.bin/markdown-toc -i README.md --bullets "*"
+yarn markdown-toc -i README.md --bullets "*"
```

Comment:
I found a cool way to run this without it needing to be installed: `npx markdown-toc -i README.md`

Reply:
We also have to decide how we want bullets to prevents back and forth between `-`s and `*`s 😆

Reply:
That's cleaner, means we could remove it from the dependency list.

Been thinking about adding in some yaml linters for a while to keep things consistent. I've got one running in VS Code, but obviously thats only marginally useful if we all end up running with different parameters.

I do wish git-hooks had a good way of modifying files in the staging area. The only solutions I've seen tend to be a bit hacky, and don't play well with the use of `git add -p`.